### PR TITLE
[WIP] Add a default tooltip if none is provided

### PR DESCRIPTION
**Pull request by ajduberstein**
#### Background

Provides a common default tooltip which can be easily overridden.

#### Change List
- Adds a default tooltip if no tooltip is provided

### CODE REVIEW

File: modules/core/src/lib/tooltip.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+//   background: 'rgba(0, 0, 0, 0.8)',
+//   color: '#fff',
+//   'max-width': '300px',
+//   'font-size': '10px',
+//   'z-index': 9,
+//   'pointer-events': 'none'
+// };
+// TODO where should the CSS be added?
+
+const buildTable = (jsonObject, shouldIndent) => {
```
Comment (ibgreen) [inline modules/core/src/lib/tooltip.js]: Nit: For big functions like this I find the traditional `function` keyword syntax easier to read. IMHO Arrow functions are best for inline functions, one line functions etc.

File: modules/core/src/lib/tooltip.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+//   padding: '4px',
+//   margin: '8px',
+//   background: 'rgba(0, 0, 0, 0.8)',
+//   color: '#fff',
+//   'max-width': '300px',
+//   'font-size': '10px',
+//   'z-index': 9,
+//   'pointer-events': 'none'
+// };
+// TODO where should the CSS be added?
```
Comment (ibgreen) [inline modules/core/src/lib/tooltip.js]: > where should the CSS be added? 

Shouldn't we just do this programatically, without need for CSS

**Comment by Pessimistress** [comment]
Can we add an RFC on how this feature will be exposed?

**Comment by ajduberstein** [comment]
Closing and will be included in jupyter-widget module

**Pull request closed without merging**